lib/helpers.js

```diff
@@ -1,12 +1,17 @@
 'use babel';
 
-import stylelint from 'stylelint';
+import path from 'path';
 import assignDeep from 'assign-deep';
 import { generateRange } from 'atom-linter';
 import presetConfig from 'stylelint-config-standard';
+import { sync as resolve } from 'resolve';
```

Comment:
Let's use the `async` version of this, just create a `Promise` wrapper around it so it can be `await`ed on.